### ISSUE TITLE
Add more information about virtual disk

### ISSUE THREAD

**Issue by aegiap** [Gandi/gandi.cli#161]
Hello,

It could be nice to have some additional information in a virtual disk details. I am thinking of:
 - if attached to a VM, show the position of the disk in the VM setup (either position 0,1,2 or directly sda,sdb,sdc,...)
 - the kernel command line associated with the disk if set.

At the moment, with gandicli **0.15**, I have:

```
$ gandi disk info sys_myvm
name      : sys_myvm
state     : created
size      : 10240
type      : data
id        : 42424242
kernel    : grub
datacenter: LU
vm        : myvm
```

I would like to see:

```
$ gandi disk info sys_myvm
name      : sys_myvm
state     : created
size      : 10240
type      : data
id        : 42424242
kernel    : grub
cmdline  : root=/dev/sda console=ttyS0 loglevel=3 
datacenter: LU
vm        : myvm 
vm_position : 0 (or sda)
```

**Comment by webcracy** [comment]
:+1:

**Comment by sayoun** [comment]
The cmdline could be added easily but not the vm_position as we don't have this information available in the API.

**Comment by sayoun** [comment]
Cmdline will be available in next release 0.17

**Comment by sayoun** [comment]
Fixed in release 0.17
